### PR TITLE
Task-35816: Add agenda space template configuration

### PR DESCRIPTION
**Pull request by AzmiTouil**
Add agenda as space default application.

### CODE REVIEW

File: agenda-webapps/src/main/webapp/WEB-INF/conf/agenda/spaces-templates-configuration.xml [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+                      <string>Agenda</string>
+                    </field>
+                    <field name="removable">
+                      <boolean>true</boolean>
+                    </field>
+                    <field name="order">
+                      <int>400</int>
+                    </field>
+                    <field name="uri">
+                      <string>Agenda</string>
```
Comment (azayati) [inline agenda-webapps/src/main/webapp/WEB-INF/conf/agenda/spaces-templates-configuration.xml]:
```suggestion
                      <string>agenda</string>
```
In all file

**Review by azayati — CHANGES_REQUESTED** [review]
See review

File: agenda-webapps/src/main/webapp/WEB-INF/conf/agenda/spaces-templates-configuration.xml (hunk 1 above)
Comment (AzmiTouil) [reply agenda-webapps/src/main/webapp/WEB-INF/conf/agenda/spaces-templates-configuration.xml]: Done !

**Review by azayati — DISMISSED** [review]
PR approved, merge waiting after cherry pick on FT branch and getting the GO from QA

File: agenda-webapps/src/main/webapp/WEB-INF/conf/agenda/spaces-templates-configuration.xml [hunk 2]
```diff
@@ -0,0 +1,196 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright (C) 2003-2010 eXo Platform SAS.
```
Comment (boubaker) [inline agenda-webapps/src/main/webapp/WEB-INF/conf/agenda/spaces-templates-configuration.xml]: 2020

Comment (AzmiTouil) [reply agenda-webapps/src/main/webapp/WEB-INF/conf/agenda/spaces-templates-configuration.xml]: Done !

**Review by azayati — APPROVED** [review]
LGTM

**Pull request merged**